man/emerge.1

```diff
@@ -570,9 +570,9 @@ ignored because any libraries that have consumers will simply be
 preserved.
 .TP
 .BR \-\-digest
-Prevent corruption from being noticed. The `repoman manifest` command is the
+Prevent corruption from being noticed. The \fBpkgdev manifest\fR command is the
 preferred way to generate manifests and it is capable of doing an entire
```

Comment:
Does `pkgdev manifest` support handling an entire repository? (I have no idea)

We should definitely replace references to `repoman`. My only question is whether the surrounding text is true of `pkgdev` as well.

Reply:
Removing manifest files from packages in my local repository followed by calling `pkgdev manifest` within the root directory of the repository resulted in generating fresh manifest files for the packages with previously removed manifests.

My conclusion is, it works in the same way `repoman` did.